yt_dlp/extractor/adn.py

```diff
@@ -134,7 +134,7 @@ def _get_subtitles(self, sub_url, video_id):
                 ssa += os.linesep + 'Dialogue: Marked=0,{},{},Default,,0,0,0,,{}{}'.format(
                     ass_subtitles_timecode(start),
                     ass_subtitles_timecode(end),
-                    '{\\a%d}' % alignment if alignment != 2 else '',
+                    f'{{\\a{alignment}}}' if alignment != 2 else '',
```

Comment:
IMO noqa this, new version is unreadable
```suggestion
                    '{\\a%d}' % alignment if alignment != 2 else '',  # noqa: UP031
```